Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerRawDataManager.swift

```diff
@@ -0,0 +1,633 @@
+//
+//  PrimerRawDataManager.swift
+//  PrimerSDK
+//
+//  Created by Evangelos on 12/7/22.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+import SafariServices
+
+@objc
+public protocol PrimerRawDataManagerDelegate {
+    @objc optional func primerRawDataManager(_ rawDataManager: PrimerHeadlessUniversalCheckout.RawDataManager, dataDidChange data: PrimerRawData?)
+    @objc optional func primerRawDataManager(_ rawDataManager: PrimerHeadlessUniversalCheckout.RawDataManager, dataIsValid isValid: Bool, errors: [Error]?)
```

Comment:
@EvansPie does the error contain specific field that has the error?

Reply:
yeah